modules/game.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../classes/game'
+require_relative '../classes/author'
+require 'json'
+module GameModule
+  DB_GAMES = './db/games.json'.freeze
+  DB_AUTHORS = './db/authors.json'.freeze
+
+  def list_all_games
+    puts "\n============ GAMES LIST =============\n\n"
+    if @games.empty?
+      puts 'Game list is empty...'
+    else
+      @games.each_with_index do |game, index|
+        puts "#{index})Multiplayer: #{game.multiplayer}
+        ,Publish Date: #{game.published_date},Last Play at:#{game.last_play_at}"
+      end
+    end
+    puts "\n"
+  end
+
+  def add_game
+    # GAME
+    puts "\n"
+    puts 'Please provide the information about the game :'
+    print 'Multilayer : '
+    multi_player = gets.chomp
+    print 'Publish Date (Ex: 2023-04-13) : '
+    published_date = gets.chomp
+    print 'Last Play Date (Ex: 2023-04-13): '
+    last_play_date = gets.chomp
+    new_game = Game.new(multi_player, last_play_date, published_date)
+    @games << new_game
+
+    # AUTHOR
+    puts "\n"
+    puts 'Please provide these information about the author:'
+    print 'First name : '
+    first_name = gets.chomp
+    print 'Last name : '
+    last_name = gets.chomp
+    new_author = Author.new(first_name, last_name)
+    @authors << new_author
+    preseve_game
```

Comment:
I kindly recommend you fix the typo on preserve to make the code more readable.

Reply:
Thank you @dfsalam for your review.